facebook.js

```diff
@@ -14,15 +14,17 @@ const emoji_map = {
   "car": "🚗",
   "child": "👦",
   "christmas tree": "🎄",
-  "closeup": "👀",
```

Comment:
This is actually "closeup" when I see it on Facebook, so "close-up" wouldn't work.